packages/template-retail-react-app/app/components/_app/index.jsx

```diff
@@ -208,6 +212,16 @@ const App = (props) => {
         }
     }, [basket?.currency])
 
+    const commerceAgentConfiguration = useMemo(() => {
```

Comment:
Nice place to prevent re-render thrashing given React component hierarchy position